### PR TITLE
[CAS-424] Add search to add DM channel view

### PR DESCRIPTION
**Pull request by adasiewiczr**
Screenshot 2020-11-26 at 11 54 40

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] Changelog updated with client-facing changes
- [ ] New code is covered by unit tests
- [x] Comparison screenshots added for visual changes
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/common/Debouncer.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+/**
+ * Utility class for debouncing high frequency events.
+ *
+ * [submit]ting a new piece of work to run within the debounce window
+ * will cancel the previously submitted pending work.
+ */
+class Debouncer(private val debounceMs: Long) {
```
Comment (adasiewiczr) [inline stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/common/Debouncer.kt:16]: Copy-paste from https://github.com/GetStream/stream-chat-android/blob/master/stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/Debouncer.kt

**Review by ogkuzmin — DISMISSED** [review]

File: stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/common/Debouncer.kt (hunk 1 above)
Comment (JcMinarro) [reply stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/common/Debouncer.kt:16]: What about to open the other one? 
Duplicate this class will be hard to maintain.

Comment (adasiewiczr) [reply stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/common/Debouncer.kt:16]: @JcMinarro we can do that, but I am not sure if we want to expose `Debouncer` to our SDK clients. What do you think?

Comment (JcMinarro) [reply stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/common/Debouncer.kt:16]: I think it is fine, it is into our own package name, so shouldn't be any issue with that

**Review by JcMinarro — APPROVED** [review]

**Review by samiuelson — APPROVED** [review]

**Pull request merged**